doc/overview/migrating.rst

```diff
@@ -28,34 +28,48 @@ below which and above which to filter out data.
 +---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
-| Filter data         | ``EEG = pop_eegfiltnew(EEG, l_freq, h_freq);``           | :func:`raw.filter(l_freq, h_freq) <mne.io.Raw.filter>`                                           |
+| Filter data         | | ``EEG = pop_eegfiltnew(EEG, l_freq, h_freq);``         | | :func:`raw.filter(l_freq, h_freq) <mne.io.Raw.filter>`                                         |
+|                     | |                                                        | |                                                                                                |
+| Common Average      | | ``EEG= pop_averef;``                                   | | :func:`raw.set_eeg_reference("average")`                                                       | 
+| referencing         | |                                                        | |                                                                                                |
 +---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
-| Run ICA             | ``EEG = pop_runica(EEG, 'pca', n);``                     | | :class:`ica = ICA(max_pca_components=n) <mne.preprocessing.ICA>`                               |
-|                     |                                                          | | :func:`ica.fit(raw) <mne.preprocessing.ICA.fit>`                                               |
+| Remove channels     | | ``pop_select.m``                                       | | :func:`raw.drop_channels()`
+|                     | |                                                        | | 
++---------------------+----------------------------------------------------------+--------------------------------------------------------------------------------------------------+
+| Run ICA             | | ``EEG = pop_runica(EEG, 'pca', n);``                   | | :func:`ica.fit(raw) <mne.preprocessing.ICA.fit>`                                               |
+|                     | |                                                        | |                                                                                                |   
+|                     | | ``EEG = pop_binica(EEG, 'pca', n);``                   | | :func:`raw = mne.preprocessing.infomax()`                                                      |
+|                     | |                                                        | |                                                                                                |
+| Plot ICA properties | | ``pop_compprop( EEG, comp_num, winhandle);``           | | :func:`ica.plot_properties(raw, picks)`                                                        |
```

Comment:
Somewhere in this table there is a rendering problem:
```
/home/circleci/project/doc/overview/migrating.rst:27: WARNING: Malformed table.
```
For this particular cell, I think you want the `|` on the left so that the entries line up properly on their lines. I would use an online renderer, starting from what renders properly on `main` and gradually adding your changes, to see what's breaking things. Something like this site (from googling "online RST rendering") would probably work -- at least it renders the `main` table okay:

https://livesphinx.herokuapp.com/

Reply:
Thank you , will work on it